lib/improver/cli/generate_topography_bands_mask.py

```diff
@@ -50,72 +43,12 @@
                    'units': 'm'}
```

Comment:
Can this dictionary be moved somewhere else? e.g: in the `generate_ancillaries` module?

Reply:
Done.

Reply:
Done.

Reply:
Thanks 👍